lab_2_keywords_cooccurrence/main.py

```diff
@@ -29,7 +64,22 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (check_list(phrases, str, False) and check_list(stop_words, str, False)):
+        return None
+    no_stop_words_list = []
+    candidate_phrases = []
```

Comment:
i don't see any references of this variable except for when it is given another value